diff -ur bacula-11.0.6.orig/src/version.h bacula-11.0.6/src/version.h
--- bacula-11.0.6.orig/src/version.h	2022-04-03 11:31:32.929298827 +0000
+++ bacula-11.0.6/src/version.h	2022-04-03 11:32:34.071438141 +0000
@@ -87,9 +87,9 @@
  *   it can always be turned on, but we advise to use it only
  *   for debug
  */
-# ifndef USE_LOCKMGR
-#  define USE_LOCKMGR
-# endif /* USE_LOCKMGR */
+//# ifndef USE_LOCKMGR
+//#  define USE_LOCKMGR
+//# endif /* USE_LOCKMGR */
 /*
  * Enable priority management with the lock manager
  *
