diff -dur bacula-5.2.13.orig/src/version.h bacula-5.2.13/src/version.h
--- bacula-5.2.13.orig/src/version.h	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/version.h	2013-12-04 20:36:09.987318325 +0100
@@ -77,9 +77,9 @@
  *   it can always be turned on, but we advise to use it only
  *   for debug
  */
-# ifndef _USE_LOCKMGR
-#  define _USE_LOCKMGR
-# endif /* _USE_LOCKMGR */
+//# ifndef _USE_LOCKMGR
+//#  define _USE_LOCKMGR
+//# endif /* _USE_LOCKMGR */
 /*
  * Enable priority management with the lock manager
  *
