--- bacula-5.0.3.orig/autoconf/configure.in.orig	2010-08-06 20:31:44.000000000 +0000
+++ bacula-5.0.3.orig/autoconf/configure.in	2010-08-06 20:36:59.000000000 +0000
@@ -1188,7 +1188,7 @@
    fi
 
    if test "$support_crypto" = "yes"; then
-      AC_CHECK_LIB(ssl, EVP_PKEY_encrypt_old, AC_DEFINE(HAVE_OPENSSLv1, 1, [Set if have OpenSSL version 1.x]))
+      AC_CHECK_LIB(crypto, EVP_PKEY_encrypt_old, AC_DEFINE(HAVE_OPENSSLv1, 1, [Set if have OpenSSL version 1.x]))
    fi
 
    LIBS="${saved_LIBS}"
