diff -dur -x '*~' bacula-7.0.5.orig/autoconf/configure.in bacula-7.0.5/autoconf/configure.in
--- bacula-7.0.5.orig/autoconf/configure.in	2014-07-29 18:31:22.000000000 +0200
+++ bacula-7.0.5/autoconf/configure.in	2015-03-22 13:46:06.462795073 +0100
@@ -3152,6 +3152,14 @@
        platforms/suse/bacula-sd \
        platforms/suse/bacula-dir"
   ;;
+pld-linux)
+   DISTNAME="PLD/Linux"
+   DISTVER=$(awk ' { print $1 " " $4  } ' /etc/pld-release 2> /dev/null)
+   TAPEDRIVE="/dev/nst0"
+   PSCMD="ps -e -o pid,command"
+   PFILES="${PFILES} \
+       platforms/unknown/Makefile"
+  ;;
 unknown)
    DISTVER=unknown
    TAPEDRIVE="/dev/nst0"
