diff -ur bacula-2.2.6.org/autoconf/bacula-macros/db.m4 bacula-2.2.6/autoconf/bacula-macros/db.m4
--- bacula-2.2.6.org/autoconf/bacula-macros/db.m4	2007-08-30 18:19:42.000000000 +0200
+++ bacula-2.2.6/autoconf/bacula-macros/db.m4	2007-11-24 18:59:45.582166513 +0100
@@ -21,12 +21,9 @@
                    MYSQL_BINDIR=/usr/local/mysql/bin
            elif test -f /usr/include/mysql/mysql.h; then
                    MYSQL_INCDIR=/usr/include/mysql
-                   if test -f /usr/lib64/mysql/libmysqlclient_r.a \
-                        -o -f /usr/lib64/mysql/libmysqlclient_r.so; then  
-                           MYSQL_LIBDIR=/usr/lib64/mysql
-                   elif test -f /usr/lib/mysql/libmysqlclient_r.a \
-                          -o -f /usr/lib/mysql/libmysqlclient_r.so; then
-                           MYSQL_LIBDIR=/usr/lib/mysql
+                   if test -f /usr/lib64/libmysqlclient_r.a \
+                        -o -f /usr/lib64/libmysqlclient_r.so; then  
+                           MYSQL_LIBDIR=/usr/lib64
                    else
                            MYSQL_LIBDIR=/usr/lib
                    fi
@@ -97,7 +94,11 @@
        AC_DEFINE(HAVE_THREAD_SAFE_MYSQL)
     fi
     SQL_BINDIR=$MYSQL_BINDIR
-    SQL_LIB=$MYSQL_LIBDIR/libmysqlclient_r.a
+    if test -f $MYSQL_LIBDIR/libmysqlclient_r.a; then
+	SQL_LIB=$MYSQL_LIBDIR/libmysqlclient_r.a
+    else
+	SQL_LIB=$MYSQL_LIBDIR/libmysqlclient_r.so
+    fi
 
     AC_DEFINE(HAVE_MYSQL)
     AC_MSG_RESULT(yes)
