diff -dur -x '*~' bacula-1.38.6.orig/src/stored/mount.c bacula-1.38.6/src/stored/mount.c
--- bacula-1.38.6.orig/src/stored/mount.c	2006-03-16 19:16:44.000000000 +0100
+++ bacula-1.38.6/src/stored/mount.c	2006-04-12 17:40:06.000000000 +0200
@@ -328,7 +328,7 @@
     *
     *  If the tape is marked as Recycle, we rewrite the label.
     */
-   if (dev->VolHdr.LabelType == PRE_LABEL || recycle) {
+   if ((dev->VolHdr.LabelType == PRE_LABEL && !dev->is_dvd()) || recycle) {
       if (!rewrite_volume_label(dcr, recycle)) {
          mark_volume_in_error(dcr);
          goto mount_next_vol;
