diff -ur bacula-11.0.6.orig/autoconf/configure.in bacula-11.0.6/autoconf/configure.in
--- bacula-11.0.6.orig/autoconf/configure.in	2022-04-03 11:29:43.383049234 +0000
+++ bacula-11.0.6/autoconf/configure.in	2022-04-03 11:30:02.645093123 +0000
@@ -3642,6 +3642,14 @@
        platforms/suse/bacula-sd \
        platforms/suse/bacula-dir"
   ;;
+pld-linux)
+   DISTNAME="PLD/Linux"
+   DISTVER=$(awk ' { print $1 " " $4  } ' /etc/pld-release 2> /dev/null)
+   TAPEDRIVE="/dev/nst0"
+   PSCMD="ps -e -o pid,command"
+   PFILES="${PFILES} \
+       platforms/unknown/Makefile"
+  ;;
 unknown)
    DISTVER=unknown
    TAPEDRIVE="/dev/nst0"
