diff -dur -x '*~' bacula-5.2.13.orig/src/baconfig.h bacula-5.2.13/src/baconfig.h
--- bacula-5.2.13.orig/src/baconfig.h	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/baconfig.h	2013-12-04 16:12:31.000000000 +0100
@@ -473,7 +473,6 @@
 #else 
 /* Groan, WxWidgets has its own way of doing NLS so cleanup */
 #ifndef ENABLE_NLS
-#undef _
 #undef setlocale
 #undef textdomain
 #undef bindtextdomain
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/authenticate.c bacula-5.2.13/src/wx-console/authenticate.c
--- bacula-5.2.13.orig/src/wx-console/authenticate.c	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/authenticate.c	2013-12-04 16:07:18.000000000 +0100
@@ -48,12 +48,13 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 #include "console_conf.h"
 #include "jcr.h"
 
-#include <wx/intl.h>
-
 #include "csprint.h"
 
 void senditf(char *fmt, ...);
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/console_conf.c bacula-5.2.13/src/wx-console/console_conf.c
--- bacula-5.2.13.orig/src/wx-console/console_conf.c	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/console_conf.c	2013-12-04 16:07:36.000000000 +0100
@@ -57,11 +57,12 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 #include "console_conf.h"
 
-#include <wx/intl.h>
-
 /* Define the first and last resource ID record
  * types. Note, these should be unique for each
  * daemon though not a requirement.
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/console_thread.cpp bacula-5.2.13/src/wx-console/console_thread.cpp
--- bacula-5.2.13.orig/src/wx-console/console_thread.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/console_thread.cpp	2013-12-04 16:07:54.000000000 +0100
@@ -46,6 +46,9 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 #include "console_conf.h"
 
@@ -53,7 +56,6 @@
 
 #include <wx/wxprec.h>
 
-#include <wx/thread.h>
 #include <wx/file.h>
 
 #include "csprint.h"
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/console_thread.h bacula-5.2.13/src/wx-console/console_thread.h
--- bacula-5.2.13.orig/src/wx-console/console_thread.h	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/console_thread.h	2013-12-04 16:08:10.000000000 +0100
@@ -37,18 +37,8 @@
 #ifndef CONSOLE_THREAD_H
 #define CONSOLE_THREAD_H
 
-#ifndef ENABLE_NLS
-#undef setlocale
-#undef textdomain
-#undef bindtextdomain
-#endif
-
-
 #include <wx/wxprec.h>
 
-#include <wx/string.h>
-#include <wx/thread.h> // inheriting class's header file
-#include <wx/intl.h> /* We need to have _ and N_ defined by wxWidgets before Bacula tries to redefine them */
 #include "bacula.h"
 #include "jcr.h"
 
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/main.cpp bacula-5.2.13/src/wx-console/main.cpp
--- bacula-5.2.13.orig/src/wx-console/main.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/main.cpp	2013-12-04 16:08:36.000000000 +0100
@@ -50,11 +50,13 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 
 #include <wx/wxprec.h>
 #include <wx/config.h>
-#include <wx/intl.h>
 #include "wxbmainframe.h"
 #include "csprint.h"
 
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/wxbconfigfileeditor.cpp bacula-5.2.13/src/wx-console/wxbconfigfileeditor.cpp
--- bacula-5.2.13.orig/src/wx-console/wxbconfigfileeditor.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/wxbconfigfileeditor.cpp	2013-12-04 16:08:55.000000000 +0100
@@ -41,6 +41,9 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 #include "wxbconfigfileeditor.h"
 #include <wx/file.h>
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/wxbconfigpanel.cpp bacula-5.2.13/src/wx-console/wxbconfigpanel.cpp
--- bacula-5.2.13.orig/src/wx-console/wxbconfigpanel.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/wxbconfigpanel.cpp	2013-12-04 16:09:13.000000000 +0100
@@ -41,6 +41,9 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 #include "wxbconfigpanel.h"
 #include <wx/arrimpl.cpp>
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/wxbhistorytextctrl.cpp bacula-5.2.13/src/wx-console/wxbhistorytextctrl.cpp
--- bacula-5.2.13.orig/src/wx-console/wxbhistorytextctrl.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/wxbhistorytextctrl.cpp	2013-12-04 16:09:43.000000000 +0100
@@ -41,6 +41,9 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 #include "wxbhistorytextctrl.h"
 
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/wxblistctrl.cpp bacula-5.2.13/src/wx-console/wxblistctrl.cpp
--- bacula-5.2.13.orig/src/wx-console/wxblistctrl.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/wxblistctrl.cpp	2013-12-04 16:10:01.000000000 +0100
@@ -42,6 +42,9 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 
 #include "wxblistctrl.h"
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/wxbmainframe.cpp bacula-5.2.13/src/wx-console/wxbmainframe.cpp
--- bacula-5.2.13.orig/src/wx-console/wxbmainframe.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/wxbmainframe.cpp	2013-12-04 16:10:26.000000000 +0100
@@ -41,6 +41,9 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 
 #include "wxbmainframe.h" // class's header file
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/wxbrestorepanel.cpp bacula-5.2.13/src/wx-console/wxbrestorepanel.cpp
--- bacula-5.2.13.orig/src/wx-console/wxbrestorepanel.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/wxbrestorepanel.cpp	2013-12-04 16:10:39.000000000 +0100
@@ -69,6 +69,9 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 
 #include "wxbrestorepanel.h"
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/wxbtableparser.cpp bacula-5.2.13/src/wx-console/wxbtableparser.cpp
--- bacula-5.2.13.orig/src/wx-console/wxbtableparser.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/wxbtableparser.cpp	2013-12-04 16:10:59.000000000 +0100
@@ -48,6 +48,9 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 
 #include "wxbtableparser.h" // class's header file
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/wxbtreectrl.cpp bacula-5.2.13/src/wx-console/wxbtreectrl.cpp
--- bacula-5.2.13.orig/src/wx-console/wxbtreectrl.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/wxbtreectrl.cpp	2013-12-04 16:11:22.000000000 +0100
@@ -42,6 +42,9 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 
 #include "wxbtreectrl.h"
diff -dur -x '*~' bacula-5.2.13.orig/src/wx-console/wxbutils.cpp bacula-5.2.13/src/wx-console/wxbutils.cpp
--- bacula-5.2.13.orig/src/wx-console/wxbutils.cpp	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/wx-console/wxbutils.cpp	2013-12-04 16:11:43.000000000 +0100
@@ -41,6 +41,9 @@
 
 #undef _DEBUG
 
+#include <wx/string.h>
+#include <wx/thread.h>
+#include <wx/intl.h>
 #include "bacula.h"
 #include "wxbutils.h"
 #include "wxbmainframe.h"
--- bacula-5.2.13.orig/src/bacula.h	2013-02-19 20:21:35.000000000 +0100
+++ bacula-5.2.13/src/bacula.h	2013-12-04 16:29:31.000000000 +0100
@@ -183,7 +183,6 @@
  */
 #ifdef HAVE_WXCONSOLE
 #undef New
-#undef _
 #undef free
 #undef malloc
 #endif
