--- bacula-3.0.0/scripts/bat.desktop.in~	2007-04-29 16:48:18.000000000 +0300
+++ bacula-3.0.0/scripts/bat.desktop.in	2009-04-15 19:11:45.310914865 +0300
@@ -1,11 +1,10 @@
 [Desktop Entry]
 Name=Bacula Administration Tool
 Comment=Bacula Director Console
-Icon=/usr/share/pixmaps/bat_icon.png
+Icon=bacula
 Exec=@sbindir@/bat -c @sysconfdir@/bat.conf
 Terminal=false
 Type=Application
 Encoding=UTF-8
 StartupNotify=true
-X-Desktop-File-Install-Version=0.3
-Categories=System;Application;Utility;X-Red-Hat-Base;
+Categories=System;Utility;
