diff -urN bacula-2.2.6.org/autoconf/bacula-macros/db.m4 bacula-2.2.6/autoconf/bacula-macros/db.m4
--- bacula-2.2.6.org/autoconf/bacula-macros/db.m4	2007-08-30 18:19:42.000000000 +0200
+++ bacula-2.2.6/autoconf/bacula-macros/db.m4	2007-11-24 18:15:52.629157615 +0100
@@ -97,8 +97,11 @@
        AC_DEFINE(HAVE_THREAD_SAFE_MYSQL)
     fi
     SQL_BINDIR=$MYSQL_BINDIR
-    SQL_LIB=$MYSQL_LIBDIR/libmysqlclient_r.a
-
+    if test -f $MYSQL_LIBDIR/libmysqlclient_r.a; then
+	SQL_LIB=$MYSQL_LIBDIR/libmysqlclient_r.a
+    else
+	SQL_LIB=$MYSQL_LIBDIR/libmysqlclient_r.so
+    fi
     AC_DEFINE(HAVE_MYSQL)
     AC_MSG_RESULT(yes)
     db_found=yes
