--- bacula-5.2.12/src/dird/bacula-dir.conf.in.orig	2012-09-13 10:51:19.000000000 +0200
+++ bacula-5.2.12/src/dird/bacula-dir.conf.in	2012-10-22 17:09:44.886778298 +0200
@@ -230,8 +230,7 @@
 # Generic catalog service
 Catalog {
   Name = MyCatalog
-# Uncomment the following line if you want the dbi driver
-@uncomment_dbi@ dbdriver = "dbi:@DEFAULT_DB_TYPE@"; dbaddress = 127.0.0.1; dbport = @db_port@
+  dbdriver = "dbi:mysql"; dbaddress = 127.0.0.1; dbport = 3306
   dbname = "@db_name@"; dbuser = "@db_user@"; dbpassword = "@db_password@"
 }
 
@@ -261,7 +260,7 @@
 #          time to time as it will grow indefinitely. However, it will
 #          also keep all your messages if they scroll off the console.
 #
-  append = "@logdir@/bacula.log" = all, !skipped
+  append = "/var/log/bacula/log" = all, !skipped
   catalog = all
 }
 
@@ -273,7 +272,7 @@
   mailcommand = "@sbindir@/bsmtp -h @smtp_host@ -f \"\(Bacula\) \<%r\>\" -s \"Bacula daemon message\" %r"
   mail = @job_email@ = all, !skipped            
   console = all, !skipped, !saved
-  append = "@logdir@/bacula.log" = all, !skipped
+  append = "/var/log/bacula/log" = all, !skipped
 }
 
 # Default pool definition
