--- bacula-1.34.1/src/lib/bsys.c.pidfile	2004-04-10 13:12:14.000000000 +0200
+++ bacula-1.34.1/src/lib/bsys.c	2004-04-18 20:04:37.000000000 +0200
@@ -299,7 +299,7 @@
    POOLMEM *fname = get_pool_memory(PM_FNAME);
    struct stat statp;
 
-   Mmsg(&fname, "%s/%s.%d.pid", dir, progname, port);
+   Mmsg(&fname, "%s/%s.pid", dir, progname);
    if (stat(mp_chr(fname), &statp) == 0) {
       /* File exists, see what we have */
       *pidbuf = 0;
@@ -343,7 +343,7 @@
       return 0;
    }
    del_pid_file_ok = FALSE;
-   Mmsg(&fname, "%s/%s.%d.pid", dir, progname, port);
+   Mmsg(&fname, "%s/%s.pid", dir, progname);
    unlink(mp_chr(fname));
    free_pool_memory(fname);
 #endif
