diff -durN bacula-5.0.1.orig/src/lib/crypto.c bacula-5.0.1/src/lib/crypto.c
--- bacula-5.0.1.orig/src/lib/crypto.c	2010-02-24 16:33:48.000000000 +0100
+++ bacula-5.0.1/src/lib/crypto.c	2010-04-21 20:50:32.587691290 +0200
@@ -49,6 +47,13 @@
 #include "jcr.h"
 #include <assert.h>
 
+/**
+ * For OpenSSL version 1.x, EVP_PKEY_encrypt no longer
+ *  exists.  It was not an official API.
+ */
+#define EVP_PKEY_encrypt EVP_PKEY_encrypt_old
+#define EVP_PKEY_decrypt EVP_PKEY_decrypt_old
+
 /*
  * Bacula ASN.1 Syntax
  *
@@ -309,7 +316,7 @@
  */
 static ASN1_OCTET_STRING *openssl_cert_keyid(X509 *cert) {
    X509_EXTENSION *ext;
-   X509V3_EXT_METHOD *method;
+   const X509V3_EXT_METHOD *method;
    ASN1_OCTET_STRING *keyid;
    int i;
 #if (OPENSSL_VERSION_NUMBER >= 0x0090800FL)
diff -durN bacula-5.0.1.orig/src/lib/tls.c bacula-5.0.1/src/lib/tls.c
--- bacula-5.0.1.orig/src/lib/tls.c	2010-02-24 16:33:48.000000000 +0100
+++ bacula-5.0.1/src/lib/tls.c	2010-04-21 20:50:32.589983312 +0200
@@ -315,7 +315,7 @@
          extname = OBJ_nid2sn(OBJ_obj2nid(X509_EXTENSION_get_object(ext)));
 
          if (strcmp(extname, "subjectAltName") == 0) {
-            X509V3_EXT_METHOD *method;
+            const X509V3_EXT_METHOD *method;
             STACK_OF(CONF_VALUE) *val;
             CONF_VALUE *nval;
             void *extstr = NULL;
