diff -dur -x '*~' bacula-5.0.2.orig/autoconf/configure.in bacula-5.0.2/autoconf/configure.in
--- bacula-5.0.2.orig/autoconf/configure.in	2010-05-26 12:46:22.437922997 +0200
+++ bacula-5.0.2/autoconf/configure.in	2010-05-26 13:08:22.688284981 +0200
@@ -1209,7 +1209,7 @@
    OPENSSL_LIBS=""
    OPENSSL_INC=""
 else
-   AC_CHECK_LIB(ssl, EVP_PKEY_encrypt_old, AC_DEFINE(HAVE_OPENSSLv1, 1, [Set if have OpenSSL version 1.x]))
+   AC_DEFINE(HAVE_OPENSSLv1, 1, [Set if have OpenSSL version 1.x])
 fi  
 
 AC_MSG_RESULT([$support_tls])
