diff -ur bacula-11.0.6.orig/scripts/mtx-changer.in bacula-11.0.6/scripts/mtx-changer.in
--- bacula-11.0.6.orig/scripts/mtx-changer.in	2022-03-10 15:57:01.000000000 +0000
+++ bacula-11.0.6/scripts/mtx-changer.in	2022-04-03 11:28:37.710899617 +0000
@@ -65,13 +65,13 @@
 myversion=2
 
 # source our conf file
-if test ! -f @scriptdir@/mtx-changer.conf ; then
+if test ! -f /etc/bacula/mtx-changer.conf ; then
   echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
-  echo "ERROR: @scriptdir@/mtx-changer.conf file not found!!!!"
+  echo "ERROR: /etc/bacula/mtx-changer.conf file not found!!!!"
   echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
   exit 1
 fi
-. @scriptdir@/mtx-changer.conf
+. /etc/bacula/mtx-changer.conf
 
 if test "${version}" != "${myversion}" ; then
   echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
