--- bacula-3.0.0.orig/Makefile.in	2009-04-22 11:34:25.143705345 +0200
+++ bacula-3.0.0/Makefile.in	2009-04-22 11:46:51.634779414 +0200
@@ -191,8 +191,8 @@
 	(cd src/cats; \
 	 chmod 755 create_bacula_database      update_bacula_tables	make_bacula_tables; \
 	 chmod 755 grant_bacula_privileges     drop_bacula_tables	drop_bacula_database; \
-	 chmod 755 create_@DB_TYPE@_database   update_@DB_TYPE@_tables	make_@DB_TYPE@_tables; \
-	 chmod 755 grant_@DB_TYPE@_privileges  drop_@DB_TYPE@_tables	drop_@DB_TYPE@_database; \
+	 chmod 755 create_*_database   update_*_tables	make_*_tables fix_*_tables; \
+	 chmod 755 grant_*_privileges  drop_*_tables	drop_*_database; \
 	 chmod 755 make_catalog_backup delete_catalog_backup)
 	(cd src/qt-console; \
 	 chmod 755 install_conf_file build-depkgs-qt-console)
--- bacula-3.0.0.orig/src/cats/Makefile.in	2009-04-22 11:34:25.204854395 +0200
+++ bacula-3.0.0/src/cats/Makefile.in	2009-04-22 12:05:18.107024138 +0200
@@ -125,19 +125,16 @@
 	$(LIBTOOL_UNINSTALL) $(RMF) $(DESTDIR)$(libdir)/libbacsql.la
 
 install: @LIBTOOL_INSTALL_TARGET@ @INCLUDE_INSTALL_TARGET@
-	$(INSTALL_SCRIPT) create_@DB_TYPE@_database $(DESTDIR)$(scriptdir)/create_@DB_TYPE@_database
-	$(INSTALL_SCRIPT) update_@DB_TYPE@_tables $(DESTDIR)$(scriptdir)/update_@DB_TYPE@_tables
-	$(INSTALL_SCRIPT) make_@DB_TYPE@_tables $(DESTDIR)$(scriptdir)/make_@DB_TYPE@_tables
-	$(INSTALL_SCRIPT) grant_@DB_TYPE@_privileges $(DESTDIR)$(scriptdir)/grant_@DB_TYPE@_privileges
-	$(INSTALL_SCRIPT) drop_@DB_TYPE@_tables $(DESTDIR)$(scriptdir)/drop_@DB_TYPE@_tables
-	$(INSTALL_SCRIPT) drop_@DB_TYPE@_database $(DESTDIR)$(scriptdir)/drop_@DB_TYPE@_database
-
-	$(INSTALL_SCRIPT) create_bacula_database $(DESTDIR)$(scriptdir)/create_bacula_database
-	$(INSTALL_SCRIPT) update_bacula_tables $(DESTDIR)$(scriptdir)/update_bacula_tables
-	$(INSTALL_SCRIPT) make_bacula_tables $(DESTDIR)$(scriptdir)/make_bacula_tables
-	$(INSTALL_SCRIPT) grant_bacula_privileges $(DESTDIR)$(scriptdir)/grant_bacula_privileges
-	$(INSTALL_SCRIPT) drop_bacula_tables $(DESTDIR)$(scriptdir)/drop_bacula_tables
-	$(INSTALL_SCRIPT) drop_bacula_database $(DESTDIR)$(scriptdir)/drop_bacula_database
+	for dbtype in bdb mysql postgresql sqlite3 sqlite; do \
+		$(INSTALL_SCRIPT) create_$${dbtype}_database $(DESTDIR)$(scriptdir)/create_$${dbtype}_database; \
+		$(INSTALL_SCRIPT) update_$${dbtype}_tables $(DESTDIR)$(scriptdir)/update_$${dbtype}_tables; \
+		$(INSTALL_SCRIPT) make_$${dbtype}_tables $(DESTDIR)$(scriptdir)/make_$${dbtype}_tables; \
+		$(INSTALL_SCRIPT) grant_$${dbtype}_privileges $(DESTDIR)$(scriptdir)/grant_$${dbtype}_privileges; \
+		$(INSTALL_SCRIPT) drop_$${dbtype}_tables $(DESTDIR)$(scriptdir)/drop_$${dbtype}_tables; \
+		$(INSTALL_SCRIPT) drop_$${dbtype}_database $(DESTDIR)$(scriptdir)/drop_$${dbtype}_database; \
+	done
+
+		$(INSTALL_SCRIPT) fix_postgresql_tables $(DESTDIR)$(scriptdir)/fix_postgresql_tables
 
 	@filename=make_catalog_backup; \
 	   if  test -f $(DESTDIR)$(scriptdir)/$$filename; then \
