diff -urN bacula-2.2.6.org/autoconf/configure.in bacula-2.2.6/autoconf/configure.in
--- bacula-2.2.6.org/autoconf/configure.in	2007-10-24 20:37:02.000000000 +0200
+++ bacula-2.2.6/autoconf/configure.in	2007-11-24 19:06:59.025825009 +0100
@@ -574,9 +574,9 @@
 
 got_conio="no"
 if test x$support_conio = xyes; then
-   AC_CHECK_HEADER(termcap.h, 
-     [ AC_CHECK_LIB(termcap, tgetent, 
-       [ CONS_LIBS="-ltermcap"
+   AC_CHECK_HEADER(ncurses.h, 
+     [ AC_CHECK_LIB(tinfo, tgetent, 
+       [ CONS_LIBS="-ltinfo"
 	 CONS_OBJ="conio.o"
 	 CONS_SRC="conio.c"
 	 got_conio="yes"
@@ -646,7 +646,7 @@
 	  fi
 	  AC_CHECK_HEADER(${with_readline}/readline.h, 
 	     [ AC_DEFINE(HAVE_READLINE, 1, [Set to enable readline support])
-	       CONS_LIBS="-lreadline -lhistory -ltermcap"
+	       CONS_LIBS="-lreadline -lhistory"
 	       got_readline="yes"   
 	     ],
 	     [ echo " "
@@ -662,14 +662,14 @@
 	 [ AC_DEFINE(HAVE_READLINE, 1, [Set to enable readline support])
 	   got_readline="yes"
 	   CONS_INC="-I/usr/include/readline"
-	   CONS_LIBS="-lreadline -ltermcap"
+	   CONS_LIBS="-lreadline"
 	 ], [
 	   # Did not find standard library, so try Bacula's default
 	    AC_CHECK_HEADER(${TOP_DIR}/depkgs/readline/readline.h, 
 	      [ AC_DEFINE(HAVE_READLINE, 1, [Set to enable readline support])
 		 got_readline="yes"   
 		 CONS_INC="-I${TOP_DIR}/depkgs/readline"
-		 CONS_LIBS="-lreadline -lhistory -ltermcap"
+		 CONS_LIBS="-lreadline -lhistory"
 		 CONS_LDFLAGS="-L${TOP_DIR}/depkgs/readline"
 		 PRTREADLINE_SRC="${TOP_DIR}/depkgs/readline"
 	      ],
