--- bacula-5.0.3/autoconf/configure.in~	2010-08-05 17:29:51.000000000 +0300
+++ bacula-5.0.3/autoconf/configure.in	2010-08-22 18:48:37.975555296 +0300
@@ -813,11 +813,12 @@
 fi
 
 TERM_LIB=""
-AC_CHECK_HEADER(curses.h, 
-  [ AC_CHECK_LIB(ncurses, tgetent, 
-    [ TERM_LIB="-lncurses" ],
-    [ AC_CHECK_LIB(termcap, tgetent, [ TERM_LIB="-ltermcap" ])
+AC_CHECK_HEADER(curses.h, [
+  AC_CHECK_LIB(tinfo, tgetent, [ TERM_LIB="-ltinfo" ], [
+   AC_CHECK_LIB(ncurses, tgetent, [ TERM_LIB="-lncurses" ], [
+    AC_CHECK_LIB(termcap, tgetent, [ TERM_LIB="-ltermcap" ])
     ])
+  ])
   ],
   [  AC_CHECK_HEADERS(curses.h)
      AC_CHECK_HEADER(term.h,
