--- bacula-5.0.1/autoconf/configure.in~	2010-03-16 15:55:32.000000000 +0100
+++ bacula-5.0.1/autoconf/configure.in	2010-03-16 15:56:00.000000000 +0100
@@ -3120,7 +3120,6 @@
    chmod 755 bat
    rm -f Makefile
    $QMAKEBIN
-   ${MAKE:-make} clean
    cd ${BUILD_DIR}
 fi
 
@@ -3194,12 +3193,6 @@
    fi
 fi
 
-# clean up any old junk
-echo " "
-echo "Cleaning up"
-echo " "
-${MAKE:-make} clean
-
 if test "x${db_type}" = "xInternal" ; then
    echo " "
    echo " "
