Index: src/stored/block.c
===================================================================
--- src/stored/block.c	(revision 2)
+++ src/stored/block.c	(working copy)
@@ -912,7 +912,7 @@
          (dev->part <= dev->VolCatInfo.VolCatParts));*/
    /* Check for part file end */
    if ((dev->num_parts > 0) &&
-        ((dev->file_addr-dev->part_start) == dev->part_size) && 
+        (dev->at_eof()) && 
         (dev->part < dev->num_parts)) {
       if (dvd_open_next_part(dcr) < 0) {
          Jmsg2(dcr->jcr, M_FATAL, 0, _("Unable to open device next part %s: ERR=%s\n"),
