diff -ur libs3-20200523/src/request.c libs3-20200523/src/request.c
--- libs3-20200523/src/request.c	2020-05-23 16:23:54.000000000 +0000
+++ libs3-20200523/src/request.c	2022-04-03 11:36:49.802020806 +0000
@@ -1744,11 +1744,6 @@
 #if LIBCURL_VERSION_NUM >= 0x073E00 /* 7.62.0 */
     case CURLE_PEER_FAILED_VERIFICATION:
 #else
-    #if LIBCURL_VERSION_NUM >= 0x071101 /* 7.17.1 */
-        case CURLE_PEER_FAILED_VERIFICATION:
-    #else
-        case CURLE_SSL_PEER_CERTIFICATE:
-    #endif
     case CURLE_SSL_CACERT:
 #endif
         return S3StatusServerFailedVerification;
