--- bacula-5.2.1/src/cats/make_catalog_backup.in~	2011-10-30 13:03:42.000000000 +0100
+++ bacula-5.2.1/src/cats/make_catalog_backup.in	2011-11-01 17:27:38.863760358 +0100
@@ -13,12 +13,15 @@
 #     line arguments and the environment using ps.  Please consult your
 #     MySQL or PostgreSQL manual for secure methods of specifying the
 #     password.
+#     In MySQL, you can create @working_dir@/.my.cnf and put password there in [client] section.
 #  $4 is the host on which the database is located
 #     (default "")
 #  $5 is the type of database
 #
 #
 
+HOME=@working_dir@
+
 default_db_type=@DEFAULT_DB_TYPE@
 
 #
