diff -ur bacula-11.0.6.orig/src/cats/make_catalog_backup.in bacula-11.0.6/src/cats/make_catalog_backup.in
--- bacula-11.0.6.orig/src/cats/make_catalog_backup.in	2022-04-03 11:30:54.303210821 +0000
+++ bacula-11.0.6/src/cats/make_catalog_backup.in	2022-04-03 11:31:21.121271928 +0000
@@ -16,12 +16,15 @@
 #     line arguments and the environment using ps.  Please consult your
 #     MySQL or PostgreSQL manual for secure methods of specifying the
 #     password.
+#     In MySQL, you can create @working_dir@/.my.cnf and put password there in [client] section.
 #  $4 is the host on which the database is located
 #     (default "")
 #  $5 is the type of database
 #
 #
 
+HOME=@working_dir@
+
 default_db_type=@DEFAULT_DB_TYPE@
 user=${2:-@db_user@}
 
