diff -ur bacula-5.2.13/src/wx-console/wxbmainframe.cpp bacula-5.2.13.wx/src/wx-console/wxbmainframe.cpp
--- bacula-5.2.13/src/wx-console/wxbmainframe.cpp	2014-01-18 20:03:40.675365632 +0100
+++ bacula-5.2.13.wx/src/wx-console/wxbmainframe.cpp	2014-01-18 20:03:00.478699256 +0100
@@ -100,7 +100,7 @@
 
 DEFINE_EVENT_TYPE(wxbTHREAD_EVENT)
 
-typedef void (wxEvtHandler::*wxThreadEventFunction)(wxbThreadEvent&);
+//typedef void (wxEvtHandler::*wxThreadEventFunction)(wxbThreadEvent&);
 
 #define EVT_THREAD_EVENT(id, fn) \
     DECLARE_EVENT_TABLE_ENTRY( \
@@ -850,7 +850,7 @@
    evt.SetEventPrintObject(po);
    
    if (wxbMainFrame::GetInstance()) {
-      wxbMainFrame::GetInstance()->AddPendingEvent(evt);
+      wxbMainFrame::GetInstance()->GetEventHandler()->AddPendingEvent(evt);
    }
 }
 
diff -ur bacula-5.2.13/src/wx-console/wxbrestorepanel.cpp bacula-5.2.13.wx/src/wx-console/wxbrestorepanel.cpp
--- bacula-5.2.13/src/wx-console/wxbrestorepanel.cpp	2014-01-18 20:03:40.675365632 +0100
+++ bacula-5.2.13.wx/src/wx-console/wxbrestorepanel.cpp	2014-01-18 20:00:54.408700169 +0100
@@ -1276,8 +1276,8 @@
       for (int i = tableparser->GetCount()-1; i > -1; i--) {
          wxString str = (*tableparser)[i][3];
          wxDateTime datetime;
-         const wxChar* chr;
-         if ( ( (chr = datetime.ParseDate(str.GetData()) ) != NULL ) && ( datetime.ParseTime(++chr) != NULL ) && ! lastdatetime.IsEqualTo(datetime) ) {
+         const wxChar* chr = datetime.ParseDate(str);
+         if ( (chr != NULL) && ( datetime.ParseTime(++chr) != NULL ) && ! lastdatetime.IsEqualTo(datetime) ) {
             lastdatetime = datetime;
             datetime += wxTimeSpan::Seconds(1);
             configPanel->AddRowChoice(_("Before"),
@@ -1937,9 +1937,9 @@
 void wxbRestorePanel::SetStatus(status_enum newstatus) {
    switch (newstatus) {
    case disabled:
-      centerSizer->Remove(configPanel);
-      centerSizer->Remove(restorePanel);
-      centerSizer->Remove(treelistPanel);
+      centerSizer->Detach(configPanel);
+      centerSizer->Detach(restorePanel);
+      centerSizer->Detach(treelistPanel);
       treelistPanel->Show(false);
       restorePanel->Show(false);
       centerSizer->Add(configPanel, 1, wxEXPAND);
@@ -1958,9 +1958,9 @@
       cancelled = 0;
       break;
    case finished:
-      centerSizer->Remove(configPanel);
-      centerSizer->Remove(restorePanel);
-      centerSizer->Remove(treelistPanel);
+      centerSizer->Detach(configPanel);
+      centerSizer->Detach(restorePanel);
+      centerSizer->Detach(treelistPanel);
       treelistPanel->Show(false);
       restorePanel->Show(false);
       centerSizer->Add(configPanel, 1, wxEXPAND);
@@ -2001,7 +2001,7 @@
    case choosing:
       start->Enable(true);
       start->SetLabel(_("Restore"));
-      centerSizer->Remove(configPanel);
+      centerSizer->Detach(configPanel);
       configPanel->Show(false);
       centerSizer->Add(treelistPanel, 1, wxEXPAND);
       treelistPanel->Show(true);
@@ -2017,7 +2017,7 @@
       configPanel->Enable(false);
       tree->Enable(false);
       list->Enable(false);
-      centerSizer->Remove(treelistPanel);
+      centerSizer->Detach(treelistPanel);
       treelistPanel->Show(false);
       centerSizer->Add(restorePanel, 1, wxEXPAND);
       restorePanel->Show(true);
